app/charts/shared/colors.tsx

```diff
@@ -18,3 +20,9 @@ export const rgbArrayToHex = (rgbArray: number[]): string => {
       );
   }
 };
+
+export const hasDimensionDefaultColors = (
+  d?: DimensionMetadataFragment
+): d is DimensionMetadataFragment => {
+  return d?.values?.[0]?.color !== undefined;
```

Comment:
If by any chance, the 1st color has no color, we have no palette. I think it would be good to be a bit more robust here and do a `some(x => x.color)`.

There is a small chance that the problem appears but it would be somewhat hard to debug I guess if it happens.

The problem would be for dimensions with a lot of values, where this check should be memoized.

Reply:
Yes, that's a really good point! I will change this behavior as suggested 🙇